js/main.js

```diff
@@ -67,7 +64,9 @@ function entryAdd(entry) {
 window.addEventListener('DOMContentLoaded', domContentLoadedListener);
 
 function domContentLoadedListener(event) {
-  if (data.nextEntryId !== 1) {
+  switchViews(data.view);
+
+  if (data.entries !== null) {
```

Comment:
This is not how to check if an array is empty, you want to check if the length is `0`